sanic/server.py

```diff
@@ -230,6 +233,18 @@ def data_received(self, data):
             exception = InvalidUsage(message)
             self.write_error(exception)
 
+        if self.is_request_stream and not self._paused and \
```

Comment:
Can we turn `self.request is not None and self.request.stream and self.is_request_stream and not self._paused and` into a property? `is_active_stream` for instance.